luxonis_train/nodes/efficientnet.py

```diff
@@ -39,4 +40,10 @@ def forward(self, x: Tensor) -> list[Tensor]:
             x = m(x)
             if i in self.out_indices:
                 outs.append(x)
+
+        x = self.backbone.conv_head(x)
+        x = self.backbone.bn2(x)
+        x = self.backbone.act2(x)
+        x = self.backbone.global_pool(x)
```

Comment:
Is there a reason we append flattened features here instead of before global pool?
It seems also we have a list of size 5?

Reply:
Whereas elsewhere it's a list of size 4?

Reply:
I think maybe we check the paper if they define "blocks" to find out which features we should take?